### DISCUSSION TITLE
Why did you stop release 7.0.1xx images after 7.0.103?

### DISCUSSION THREAD

**Discussion by MattBussing** [dotnet/dotnet-docker#4596] [Q&A]
Why did you stop release 7.0.1xx images after 7.0.103? There is an issue with 7.0.1xx images and dotnet format until 7.0.104 and greater. https://github.com/dotnet/format/issues/1800. So, I'd like to be able to fix that with a 7.0.105 image.

**Comment by lbussell** [comment]
Hi @MattBussing,

> Why did you stop release 7.0.1xx images after 7.0.103?

We only release .NET container images for the latest available feature band for a given SDK major version. We stopped building 7.0.1xx images when 7.0.2xx was released and stable: https://github.com/dotnet/dotnet-docker/pull/4452. For 7.0, that means the latest release is now 7.0.203. I filed https://github.com/dotnet/dotnet-docker/issues/4597 to formally document this policy since I couldn't find it written down anywhere.

As for the dotnet format problems, according to the issue you linked it looks like it still affects 7.0.203. I would expect the issue to be resolved when 7.0.204 is released. Unfortunately I don't think we should ship a one-off 7.0.105 image just to address the dotnet-format issue, so I would encourage you to try any of the workarounds posted in the issues you linked.
